### PR TITLE
Add a reminder to have Restricted or at-since in new public stuff

### PR DESCRIPTION
**Pull request by daniel-beck**

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
